packages/search/README.md

```diff
@@ -55,6 +55,117 @@ const { agoSearch } = require('@esri/hub-search')
+  access: "private",
+  culture: ["en", "de"],
+  categories: {
+    value: ["category one", "category 2", "category three"],
+    bool: "AND"
+  }
+};
+```
+
+The `options` property provides several familar options for sorting and paging data, as well as requesting aggregations. These are specified by the `IContentSearchOptions` interface. Importantly, a user can also provide an `UserSession` instance as part of the options for authenticated searches. For the `searchContent` function, the client must provide the UserSession instance, `portal` sharing url, and `isPortal` flag, as needed, for each invocation. This is because it is a standalone function with no default options to reference. By contrast, the `ContentSearchService` will use the options it was configured with as a default if the relevant option is not provided.
+
+#### Content Search Results
+The results of a content search could include:
+  - `results`: a list of content returned from the search
+  - `count`: the number of results returned from the request
+  - `total`: the total number of results that will be returned from multiple paginated searches
+  - `query`: the stringified query used to complete the search,
+  - `aggregations`: any aggregations returned
+  - `hasNext`: a boolean indicating if there are more pages of results
+  - `next`: a function that, when invoked, will fetch the next page of results
+
+Importantly, one can optionally provide a UserSession instance to the `next` function invocation to be used for authenticated searches
+
+```
+const nextResponse = await firstResponse.next(userSession);
+```
+
+#### Known caveats with the V3 Search API
+There are a few caveats concerning the V3 Search API. If these are an issue, one still should be able to instantiate the `ContentSearchService` with `isPortal: true` to limit searching to the Portal API, or simply query the API directly.
+
+1. Multiple cross-field searching is not currently possible.
+Single cross-field searching is possible. For example, one can search for any public Feature Services.
+`(type: "Feature Service" AND access: public)`
+
+However, multiple cross-field searching is not.
+`(type: "Feature Service" AND access: public) OR (type: "Table" AND access: private)`
+
+2. Fuzzy searching cannot be field-specific
+The Portal API allows fuzzy searching via a search term or on specific properties such as `title` and `description`. For example `(title: "Brampton")` will perform a search for "Brampton" in the title. By contrast, V3 Search API only provides a search term that is used across many different properties such as `searchDescription`.
+
+**Importantly, this means that, when searching the V3 Search API, supplying filter values for these fields will be interpeted as an exact search, leading to very few results. Fuzzy searching should be done via the search term**
+
+3. NOTs cannot be combined with non-NOT operators on the same field
+`(tag: TAG_ONE AND NOT tag: TAG_TWO)`
```

Comment:
Is there a way to accomplish this though? like `(tag: TAG_ONE AND -tag: TAG_TWO)` Not that we do this much with tags, but we regularly do snarly stuff like this with `typekeywords` - granted that's usually in an item picker, vs doing things with "content" per-se, but I know @ajturner has a expressed vision (at least in the past) where even item pickers are powered by the "Hub Search".

Reply:
Unfortunately not with the current API. The request syntax is very different from Portal and doesn't lend to this or the other types of searches mentioned in the caveats section. In researching search across the application, I mentioned this to @thomas-hervey, particularly regarding the `typekeywords` you mention. A really good example of searches that the V3 API cannot current support are found in the "Add Existing Content" modal.

Reply:
Ok - good to know that until this is addressed, Item Pickers should likely be powered by the Portal API directly.

Reply:
I have to agree, although I have to discuss with @thomas-hervey a bit more, particularly as we begin to add other sources such as Event Feature Layers to the search